modules/MIPS/truncate_reports.php

```diff
@@ -1,43 +1,63 @@
 <?php
-/*
- * Truncate old MIPS reports
- *
- * Copyright (C) 2015 - 2017      Suncoast Connection
- * 
- * LICENSE: This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0
- * See the Mozilla Public License for more details. 
- * If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
- * 
- * @author  Art Eaton <art@suncoastconnection.com>
- * @author  Bryan lee <leebc@suncoastconnection.com>
- * @package LibreHealthEHR 
- * @link    http://suncoastconnection.com
- * @link    http://librehealth.io
- *
- * Please support this product by sharing your changes with the LibreHealth.io community.
- */
-require_once '../../interface/globals.php';
-include_once("$srcdir/api.inc");
- include_once("$srcdir/acl.inc");
+	/*
+	 * Truncate old MIPS reports
+	 *
+	 * Copyright (C) 2015 - 2017      Suncoast Connection
+	 * 
+	 * LICENSE: This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0
+	 * See the Mozilla Public License for more details. 
+	 * If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+	 * 
+	 * @author  Art Eaton <art@suncoastconnection.com>
+	 * @author  Bryan lee <leebc@suncoastconnection.com>
+	 * @package LibreHealthEHR 
+	 * @link    http://suncoastconnection.com
+	 * @link    http://librehealth.io
+	 *
+	 * Please support this product by sharing your changes with the LibreHealth.io community.
+	 * 
+	 * IMPORTANT:
+	 * This whole feature needs to be moved to a integrated admin form, and naturally use css buttons and translations.
+	 */
+	require_once '../../interface/globals.php';
+	include_once("$srcdir/api.inc");
+	include_once("$srcdir/acl.inc");
+
+	// Warning message created here and echoed on its place. The single ' is shown as ` don't know why. This must be fixed.
+	$reportDeleteWarning = xlt("This will delete all old reports and it's irreversible!");
```

Comment:
"This will irreversibly delete all existing reports!"

Reply:
done